source/_patterns/00-atoms/components/mustard-cutting-js.mustache

```diff
@@ -0,0 +1,14 @@
+<script>
+  if (
+      window.localStorage &&
+      document.querySelector &&
+      window.addEventListener &&
+      !!(document.createElement('div')).dataset)
+  {
+    {{#scriptLoader}}
+      var $body = document.querySelector('body');
+      {{{scriptLoader}}}
```

Comment:
I think implementers should just give a list of scripts to load, rather than the actual JS.

Reply:
Maybe on `window.elifeConfig`? That way this loader can be a normal script rather than a pattern?